src/Microsoft.DotNet.Wpf/src/WindowsBase/System/Windows/Input/KeyConverter.cs

```diff
@@ -142,7 +142,7 @@ public override object ConvertTo(ITypeDescriptorContext context, CultureInfo cul
 
         private object GetKey(string keyToken, CultureInfo culture)
         {
-            if (keyToken == String.Empty)
+            if (keyToken.Length == 0)
```

Comment:
NullReferenceException?

Could you use string.IsNullOrEmpty ?

Reply:
The caller code is:

```
                string fullName = ((string)source).Trim();
                object key = GetKey(fullName, CultureInfo.InvariantCulture);
```

Trim won't return null, so `fullName` is going to be a non-null value.

Reply:
If you looked at the old code and traced it if the passed `keyToken` is null, You'll find that it will throw NullReferenceException at:

```
                keyToken = keyToken.ToUpper(culture);
```

But, anyway the passed value isn't null. So, it's okay.